### PR TITLE
ft: add sending of link to buyers cart

### PR DESCRIPTION
**Pull request by tolls-3**
The addCart controller will also be used to send an email notification of the link to a buyers cart to the buyers provided email address.

## Description
As a buyer I want to be able to save items in my cart so that I can checkout with only those items I wish to purchase at a later date.
## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/a56KYsVe/105-send-an-email-to-the-buyer-when-he-she-saves-his-her-cart

### CODE REVIEW

**Comment by tolls-3** [comment]
[Send an email to the buyer when he/she saves his/her cart](https://trello.com/c/a56KYsVe/105-send-an-email-to-the-buyer-when-he-she-saves-his-her-cart)

File: controllers/cart/addCart.js [hunk 1]
```diff
@@ -19,7 +21,20 @@ async function addCart(req, res) {
...
+      to: `${result.email}`,
+      from: process.env.FROM_EMAIL,
+      subject: 'Pure Retail email notification for your saved cart',
+      text: 'This link will lead you to your saved cart',
+      html: `<strong><a href=${link}>Click me</a></strong>`
+    }
+    await sgMail.send(msg)
+
+    res.status(200).json({
+      status: 'A reset phone has been sent to ' + result.email + '.',
```
Comment (aapetsi) [inline controllers/cart/addCart.js]: i think the status message needs to be restructured. should probably read `A reset email has been sent ...`

**Review by aapetsi — COMMENTED** [review]
Looks good. Just a minor change on the email status message

File: controllers/cart/addCart.js (hunk 1 above)
Comment (tolls-3) [reply controllers/cart/addCart.js]: Ok, I have made some changes to the status message.

**Review by aapetsi — APPROVED** [review]
All good now

**Pull request merged**